python/cli/data_platform_workflows_cli/craft_tools/collect_bases.py

```diff
@@ -36,7 +36,7 @@ def get_bases(*, craft_: craft.Craft, yaml_data):
 
     For snaps & rocks, the Ubuntu version is the same for all architectures.
     """
-    if craft_ is craft.Craft.ROCK:
+    if craft_ is craft.Craft.ROCK or "platforms" in yaml_data:
```

Comment:
**suggestion (non-blocking)**: I got a little worried of the fact that we may be looking for this string in the whole document. Turns out that `yaml_data` is a dict as this point. Would you mind adding a type hint in the signature?